tests/typechecker/array_negative_c_int.jakt

```diff
@@ -0,0 +1,11 @@
+/// Expect:
+/// - output: "-1\n2\n"
+
+function main() {
+    let array: [c_int] = [-1 as! c_int, 2 as! c_int]
+
+    if array[0] == -1 as! c_int {
```

Comment:
I don't understand why this is needed. Just print `array[0]` unconditionally...?